### ISSUE TITLE
[crash] unwrap panic while parsing wasm module with incorrect version

### ISSUE THREAD

**Issue by pventuzelo** [rhysd/wain#29]
Hey,

When providing this module to wain parser, I triggered a panic due to unwrapping on `Result::Err` variant.

``` sh
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: TryFromSliceError(())', /home/scop/.cargo/registry/src/github.com-1ecc6299db9ec823/wain-syntax-binary-0.1.3/src/parser.rs:250:61
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Download: [crash_wain_unwrap.zip](https://github.com/rhysd/wain/files/4769519/crash_wain_unwrap.zip)

**Comment by rhysd** [comment]
I could reproduce this. Thank you for reporting.

**Comment by rhysd** [comment]
I fixed it at 655e0c0. Now an error is caused properly:

```
> cargo run crash_wain_unwrap.wasm
    Finished dev [unoptimized + debuginfo] target(s) in 0.01s
     Running `target/debug/wain crash_wain_unwrap.wasm`
Error on parsing: expected version [1 0 0 0] but got [1, 49, 57, 0] while parsing module. caused at byte offset 4

 ... 01 31 39 00 61 73 6d 01 31 39 35 01
     ^
     starts from here
```

I think I should setup fuzz test.
